ui/src/components/ripple-out/RippleOutGivingCard.js

```diff
@@ -0,0 +1,217 @@
+import * as React from 'react';
+import {
+  Box,
+  Button,
+  Heading,
+  Text,
+  Link,
+  Spacer,
+  useDisclosure,
+  VStack,
+  Flex,
+  Stack,
+  Modal,
+  ModalOverlay,
+  ModalContent,
+  ModalHeader,
+  ModalBody,
+  ModalCloseButton,
+  Center,
+  Image,
+  ModalFooter,
+} from '@chakra-ui/react';
+const RenderSwitch = (param) => {
+  switch (param.ModalSelection) {
+    case 'FPS':
+      return (
+        <Flex direction={['column', 'column', 'row']}>
+          <Stack>
+            <Text fontWeight="bold">
+              Open your mobile banking FPS interface and scan the QR code OR
+              input the FPS identifier.
+            </Text>
+            <Box>
+              <Text fontWeight="bold">
+                Name: HARVEST MISSION COMMUNITY CHURCH
+              </Text>
+              <Text fontWeight="bold">FPS ID: 167534304</Text>
+            </Box>
+          </Stack>
+          <Spacer />
+
+          <Image
+            margin="auto"
+            src={process.env.PUBLIC_URL + '/images/giving/FPSQR.png'}
+            boxSize={['70%', '70%', '30%']}
+          />
+        </Flex>
+      );
+    case 'Bank Transfer':
+      return (
+        <VStack spacing="3vh">
+          <Box py="1em">
+            <Text fontWeight="bold">
+              You may use the following information:
+            </Text>
+            <Text fontWeight="bold">
+              Bank Name: China Construction Bank (Asia) Corporation Limited
+            </Text>
+            <Text fontWeight="bold">Bank Code: 009</Text>
+            <Text fontWeight="bold">Branch Code: 845</Text>
+            <Text fontWeight="bold">
+              Account Name: Harvest Mission Community Church (Hong Kong) Limited
+            </Text>
+            <Text fontWeight="bold"> Account Number: 013012090</Text>
+            <Text fontWeight="bold">
+              Transfer Remarks: Please indicate the giving is for the “Ripple
+              Out Campaign”
```

Comment:
can add example? isnt it supposed to be Ripple Out: [Your Name]? lol it kept changing